doc/source/examples/multiphysics/rising_bubble/rising_bubble.rst

```diff
@@ -1,5 +1,5 @@
 ==========================
-Rising bubble VOF
+Rising bubble
```

Comment:
The first image of this example needs to be replaced by an image with white background because when the documentation is in dark mode, it is not possible to read the numbers.